cdap-client/src/main/java/io/cdap/cdap/client/ProgramClient.java

```diff
@@ -571,10 +571,10 @@ public String getProgramLogs(ProgramId program, long start, long stop)
    */
   public Map<String, String> getRuntimeArgs(ProgramId program)
     throws IOException, UnauthenticatedException, ProgramNotFoundException, UnauthorizedException {
-
-    String path = String.format("apps/%s/versions/%s/%s/%s/runtimeargs",
-                                program.getApplication(), program.getVersion(),
-                                program.getType().getCategoryName(), program.getProgram());
+    // Introduced in LCM: apps created have versions assocaited with it and the program-id passed in the test points
```

Comment:
remove this comment, it's more related to how the callers are using this vs the implementation here. Similar comment here, it's better to make the client take ProgramReference instead of ProgramId, but we can punt that for later.